app/plugins.go

```diff
@@ -338,10 +324,15 @@ func (a *App) InitPlugins(pluginPath, webappPath string) {
 		return
 	}
 
+	err = os.Mkdir(webappPath, 0744)
+	if err != nil && !os.IsExist(err) {
+		l4g.Error("failed to start up plugins: " + err.Error())
+		return
+	}
+
 	a.PluginEnv, err = pluginenv.New(
 		pluginenv.SearchPath(pluginPath),
-		pluginenv.WebappPath(webappPath),
-		pluginenv.APIProvider(func(m *model.Manifest) (plugin.API, error) {
+		pluginenv.WebappPath(webappPath), pluginenv.APIProvider(func(m *model.Manifest) (plugin.API, error) {
```

Comment:
Was deleting this newline intentional?

Reply:
Hmmm no. I was messing around with my go formatter though so it might have done it accidentally. I'll revert